### ISSUE TITLE
Evaluation

### ISSUE THREAD

**Issue by joh-fischer** [joh-fischer/pytorch-vision#4]
Create evaluation pipeline from csv file.

- [x] load csv
- [x] group per epoch
- [x] plot train/val accuracy
- [x] plot train/val loss
- [x] show example classifications
